styles/onedark.css

```diff
@@ -0,0 +1,175 @@
+/**
```

Comment:
We can definitely choose another theme. That's the one that the finder project uses as well. Feel free to change that.

Reply:
It is good to have it consistent, but like you mentioned, we can always change it later